cmd/exchaind/repair_data.go

```diff
@@ -50,10 +54,20 @@ func repairStateCmd(ctx *server.Context) *cobra.Command {
 	cmd.Flags().String(sdk.FlagDBBackend, tmtypes.DBBackend, "Database backend: goleveldb | rocksdb")
 	cmd.Flags().Bool(sdk.FlagMultiCache, true, "Enable multi cache")
 	cmd.Flags().StringP(pprofAddrFlag, "p", "0.0.0.0:6060", "Address and port of pprof HTTP server listening")
+	cmd.Flags().Bool(flagRepairFastStorage, false, "repair fast storage")
+	cmd.Flags().MarkHidden(flagRepairFastStorage)
```

Comment:
I don't think this flag needs to be hidden

Reply:
just want use it in a small range first

Reply:
remove this flag， add a discard-fast-storage flag. user will repair fast-storage by default